src/components/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Navbar.css';
+
+function Navbar() {
+  const links = [{
+    id: 1,
+    path: '/',
+    text: 'Home',
+  },
+  {
+    id: 1,
+    path: '/calc',
+    text: 'Calculator',
+  },
+  {
+    id: 1,
+    path: '/quote',
+    text: 'Quote',
+  }];
```

Comment:
- [ ] (OPTIONAL) Kindly provide different `ids` for `links` so as to fix the error on the console of your site 👍 
- Console error:
![Annotation 2022-06-01 092000](https://user-images.githubusercontent.com/85113567/171360171-5745061e-bb9f-4bb1-a215-5d4b90e8bb42.jpg)